api/nodes/serializers.py

```diff
@@ -373,3 +379,29 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeInstitutionRelationshipSerializer(JSONAPIRelationshipSerializer):
```

Comment:
I would add id and type fields as required fields to this serializer to take advantage of some of the code we have.  `id` should be a CharField since the IDField would try to match node id to institution id.  `type` should be a TypeField.   This TypeField will take care of the 'institution' check.